aws-auth-cognito/src/main/java/com/amplifyframework/auth/cognito/actions/SRPCognitoActions.kt

```diff
@@ -122,38 +114,18 @@ object SRPCognitoActions : SRPActions {
                     challengeResponses = challengeParams
                     encodedContextData?.let { userContextData { encodedData = it } }
                 }
-                response?.let { respondToAuthChallengeResponse ->
-                    respondToAuthChallengeResponse.authenticationResult?.newDeviceMetadata?.let { deviceMetaData ->
-                        val confirmDeviceParams = DeviceMetaData(
-                            idToken = respondToAuthChallengeResponse.authenticationResult?.idToken
-                                ?: throw AuthException("Sign in failed", AuthException.TODO_RECOVERY_SUGGESTION),
-                            refreshToken = respondToAuthChallengeResponse.authenticationResult?.refreshToken
-                                ?: throw AuthException("Sign in failed", AuthException.TODO_RECOVERY_SUGGESTION),
-                            accessToken = respondToAuthChallengeResponse.authenticationResult?.accessToken
-                                ?: throw AuthException("Sign in failed", AuthException.TODO_RECOVERY_SUGGESTION),
```

Comment:
This change can not be intentional?

Reply:
It is. This handling is already done in SignInChallengeHelper. This was duplicate.